indico/web/templates/forms/date_widget.html

```diff
@@ -1,12 +1,32 @@
 {% extends 'forms/base_widget.html' %}
 
 {% block html %}
-    <input type="text" id="{{ field.id }}" value="{{ field._value() }}" name="{{ field.name }}"
-           placeholder="DD/MM/YYYY" {{ input_args | html_params }}>
+    <div class="datetime-widget i-form-field-fixed-width">
+        {# the hidden field has autofocus on purpose to prevent our JS from focusing the date picker #}
+        <input type="hidden" name="{{ field.name }}" id="{{ field.id }}-datestorage" autofocus
+               {% if field.data and not field.date_missing %}value="{{ field.data|format_date('code') }}"{% endif %}>
```

Comment:
We need to keep `{{ input_args | html_params }}` on the field, otherwise validators like HiddenUnless don't work anymore (can be tested in the clone with pattern dialog or when creating a reminder) since this adds the data attributes enabling the HiddenUnless logic.

The same bug probably exists in the datetime variant, but I guess we never use this with HiddenUnless and thus didn't notice it there.